integration_test/cleanup_containers.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-docker stop amoc-1 amoc-2 amoc-3 amoc-4 graphite
+docker stop amoc-1 amoc-2 amoc-3 amoc-4 graphite grafana
```

Comment:
amoc-4 doesn't seem to be started in `start_demo_cluster.sh` btw, I've had docker warnings telling me it couldn't remove it. Maybe it's started elsewhere, if it's not then we could just remove it from here.

Reply:
this warning is ignored. it's just a general script that cleans up containers used for integration tests or demo cluster. so it contains the whole list of them.